.travis.yml

```diff
@@ -6,27 +6,26 @@ python:
   - 3.7
 
 before_install:
-  # Install node.js following instructions from
-  #   https://nodejs.org/en/download/package-manager/
-  - sudo apt install curl
-  - curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
-  - sudo apt install nodejs
+  # Use the pre-installed nvm version to update node
+  # Apparently Travis comes with nvm even on the python containers
+  - nvm install v12.18.1
+  - nvm use v12.18.1
+  - node -v
+  - npm version
 
 install:
-  # Install node.js dependencies
-  - npm install
-  # Install python dependencies
-  - pip install -r requirements-dev.txt
+  # Install package, while updating dependencies
+  - pip install --upgrade --upgrade-strategy eager .[dev]
 
 script:
   # Run all pytest unit tests
-  - python -m pytest -v tests --cov readabilipy --cov-report term-missing --benchmark-disable
+  - cd tests && python -m pytest -v . --cov readabilipy --cov-report term-missing --benchmark-disable && cd ..
   # Run pyflakes for error detection
   - pyflakes *.py readabilipy tests
   # Check PEP8 compliance (ignoring long lines)
   - pycodestyle --statistics --ignore=E501 --count *.py readabilipy tests
   # Run pylint for stricter error checking
-  - pylint readabilipy tests
+  - pylint readabilipy
```

Comment:
Why not run pylint on the tests?

Reply:
Good catch!

Reply:
Actually I remember now, pylint gives an error for a missing __init__.py in the test directory, which apparently is a [known issue](https://stackoverflow.com/questions/48024049/pylint-raises-error-if-directory-doesnt-contain-init-py-file) with pylint. For some reason I'm not able to reproduce this error locally though, so I'll have to do a bit more digging on this issue.

Reply:
Resolved by 4d48849